### DISCUSSION TITLE
Failed to read json file, invalid escaped character in string

### DISCUSSION THREAD

**Discussion by Shinena1998** [duckdb/duckdb#14147] [Q&A]
hi.

the json file I read has the following lines
{"remote_addr":"\x00\x00\x00\x00\x00\x00\x0012.11.122.139","remote_user":"-","time_local":"[27/Sep/2024:14:57:52 +0700]"}

I hope duckdb can read remote_addr in raw string format instead of trying to escape it and reporting the error "invalid escaped character in string" . 

How can I make duckdb not report an error and read the jon file normally?

Thank you very much

**Comment by zmajeed** [comment]
JSON doesn't support byte hex escapes `\x00` in strings - https://www.json.org - only Unicode hex escapes `\u0000` are allowed - the following valid JSON should work

```json
{
  "remote_addr": "\u0000\u0000\u0000\u0000\u0000\u0000\u000012.11.122.139",
  "remote_user": "-",
  "time_local": "[27/Sep/2024:14:57:52 +0700]"
}
```
